Form-Controls/index.html

```diff
@@ -12,7 +12,45 @@
       <h1>Product Pick</h1>
     </header>
     <main>
-      <form>
+      <form>  
+        <div>
+        <label for="name">Name:</label>
+          <input type="text" id="name" required>
+        </div>  
+      <br>
+        <div>
+          <label for="email">Email:</label>
+          <input type="email" name="email" id="email" required>
+       </div>
+       <br>
+       <div>
+        <label for="colours">Choose a colour:</label>
+        <select id="colours">
```

Comment:
You are missing one important attribute in your `<select>` tag. Please think which one.

Reply:
The same question for other select tag in your code.

Reply:
Please, take a look at the comment above. It is still not fixed.